src/main/java/com/esri/samples/featurelayers/feature_layer_extrusion/FeatureLayerExtrusionSample.java

```diff
@@ -0,0 +1,146 @@
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.featurelayers.feature_layer_extrusion;
+
+import java.util.Arrays;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.data.QueryParameters;
+import com.esri.arcgisruntime.data.ServiceFeatureTable;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.OrbitLocationCameraController;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.symbology.Renderer;
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.symbology.SimpleRenderer;
+
+public class FeatureLayerExtrusionSample extends Application {
+
+  private boolean showTotalPopulation = true;
+  private SceneView sceneView;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    StackPane stackPane = new StackPane();
+    Scene fxScene = new Scene(stackPane);
+    // for adding styling to any controls that are added 
+    fxScene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+    // set title, size, and add scene to stage
+    stage.setTitle("Feature Layer Extrusion Sample");
+    stage.setWidth(800);
+    stage.setHeight(700);
+    stage.setScene(fxScene);
+    stage.show();
+
+    // so scene can be visible within application
+    ArcGISScene scene = new ArcGISScene(Basemap.createTopographic());
+    sceneView = new SceneView();
+    sceneView.setArcGISScene(scene);
+    stackPane.getChildren().add(sceneView);
+
+    // get us census data as a service feature table
+    ServiceFeatureTable statesServiceFeatureTable = new ServiceFeatureTable("http://sampleserver6.arcgisonline.com/arcgis/rest/services/Census/MapServer/3");
+    statesServiceFeatureTable.setFeatureRequestMode(ServiceFeatureTable.FeatureRequestMode.MANUAL_CACHE);
+
+    // load all features, with all fields (attributes), from service feature table
+    QueryParameters queryParams = new QueryParameters();
+    queryParams.setWhereClause("1=1");
```

Comment:
I prefer `true` here if that works. More explicit

Reply:
I tried putting `true` in there but got nothing back.